systemtests/src/main/java/io/enmasse/systemtest/apiclients/ApiClient.java

```diff
@@ -67,6 +67,11 @@ protected void reconnect() {
         connect();
     }
 
+    public void close() {
+        this.client.close();
```

Comment:
We'll NPE here if the client has not called connect(), which is likely to disguise a true failure. Please add a null check.

Reply:
Fixed